do.sh

```diff
@@ -281,6 +281,19 @@ e2e_lint() {
   fi
 }
 
+e2e_build_docs() {
+  e2e_build_templates
+  # Building JS-Dossier requires Facebook Buck (http://buckbuild.com/)
+  # It's not part of a default install.
```

Comment:
Do we expect people to install buck themselves before running ./do.sh build_docs ?